gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Tue Jun 28 11:19:41 CEST 2016
+#Mon Oct 02 13:25:01 CEST 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.14-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.2-bin.zip
```

Comment:
Can you please run something like `./gradlew --version` on this branch? 

It should update `gradle/wrapper/gradle-wrapper.jar` and `gradlew` and you need to commit them

Reply:
I did exactly that.

Reply:
Looks like 4.2.1 came out just now.

Reply:
Well yes, but it still should have updated wrapper jar and bash script hm

Reply:
Sure.